### DISCUSSION TITLE
MathML: should the app activate MathJax automatically if indicated in the OPF?

### DISCUSSION THREAD

**Discussion by llemeurfr** [edrlab/thorium-reader#1841] [Developer corner]
`mathml` is required to be present in the list of properties in the manifest <item> element of the OPF file. 
This could trigger the activation of the MathML option. 

The manuel MathML setting would still be present in user config.

**Comment by danielweck** [comment]
MathJax is intentionally an "opt-in" feature in Thorium, due to its undesirable side effects (notably, we force scroll mode when in paginated mode, in order to work around focus / autoscroll issues introduced by MathJax's own user interface).

When Thorium users decide to activate MathML support, Thorium detects the presence of Math markup in loaded HTML documents, and only injects the MathJax library if necessary. Thorium also checks that the MathJax library isn't already used in authored documents, so as to not inject its own potentially conflicting version.
